spec87ascii.json

```diff
@@ -0,0 +1,549 @@
+{
+	"name": "ISO 8583 v1987 ASCII",
+	"fields": {
+		"Field000": {
```

Comment:
I think that it would be great if we can follow go spec as close as possible. Here, instead of "Field000", I would jsut use it as "0", "1", "2", ... "32". Without "Field" prefix.

Reply:
sure, we can use like this. But the naming isn't fit for XML format. 
xml tag as number is difficult to use

Reply:
Hm... I didn't thought about XML. Maybe we can keep it without Field in JSON and use field for XML only? I don't know how many XML users we are going to have...